nephos/fabric/crypto.py

```diff
@@ -326,6 +332,70 @@ def cacerts_to_secrets(namespace, msp_path, user):
         item_to_secret(namespace, msp_path, user, item)
 
 
+def tls_to_secrets(namespace, tls_path, username):
+    """Get the tls server.crt and server.key and create the relevant secret
```

Comment:
@inzamam-iqbal, please capitalise "TLS" in comments, since it corresponds to Transport Layer Security